src/components/MapView/DateSelector/index.tsx

```diff
@@ -1,31 +1,39 @@
-import React, { forwardRef, Fragment, Ref, useEffect, useState } from 'react';
+import React, { forwardRef, Ref, useRef, useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import moment from 'moment';
+import Moment from 'moment';
+import { extendMoment } from 'moment-range';
 import {
   Button,
   createStyles,
-  Divider,
+  Fade,
   Grid,
+  Hidden,
   Theme,
+  Tooltip,
+  Typography,
   WithStyles,
   withStyles,
 } from '@material-ui/core';
 import DatePicker from 'react-datepicker';
+import Draggable from 'react-draggable';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import {
   faAngleDoubleLeft,
   faAngleDoubleRight,
 } from '@fortawesome/free-solid-svg-icons';
 import 'react-datepicker/dist/react-datepicker.css';
+import { findIndex, get, isEqual } from 'lodash';
 import { updateDateRange } from '../../../context/mapStateSlice';
-import { getMonthStartAndEnd, isAvailableMonth, months } from './utils';
+import { findDateIndex } from './utils';
 import { dateRangeSelector } from '../../../context/mapStateSlice/selectors';
 
 interface InputProps {
   value?: string;
   onClick?: () => void;
 }
 
+const moment = extendMoment(Moment as any);
```

Comment:
Maybe TS has a point complaining about this? Since from the TS typing, importing moment as `import * as moment` makes the error go away. Is the `extendMoment` working without `import  * as moment`?